src/assets/tailwind.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
Any custom styles should be added to this file HOWEVER do not add styles that tailwind already provides!